app/sidekiq/form526_submission_failure_email_job.rb

```diff
@@ -58,9 +58,9 @@ class Form526SubmissionFailureEmailJob
     raise e
   end
 
-  def perform(submission_id, date_of_failure = Time.now.utc)
+  def perform(submission_id, date_of_failure = Time.now.utc.to_s)
     @submission = Form526Submission.find(submission_id)
-    @date_of_failure = date_of_failure
+    @date_of_failure = Time.new(date_of_failure)
```

Comment:
What if you instead used `Time.parse` to create a `Time` object from the `date_of_failure` string?